cps/CP112.md

```diff
@@ -71,4 +71,4 @@ CFG wallet: `4fpQvcAb5oBvosEXo1w78SzdasuD8uCMyTU3eWaYtznJdSFg` , USDT (ERC20): `
 
 Link to the RFC on the Forum: https://gov.centrifuge.io/t/rfc-integration-of-the-network-into-tangem-and-production-of-500-branded-centrifuge-wallets-2-card-sets/6332
 
-Link to the onchain vote: 
+Link to the onchain vote: https://centrifuge.subsquare.io/treasury/proposals/2

```

Comment:
You don't want to link to the referendum vote instead?

https://centrifuge.subsquare.io/democracy/referenda/56